### ISSUE TITLE
Add command line parameter to automatically start game server(s)

### ISSUE THREAD

**Issue by Joe-Klauza** [Joe-Klauza/sandstorm-admin-wrapper#24]
For automation situations (like the newly added systemd example script), we'll want a command-line option to automatically start the server. Once #23 is implemented, we can have a repeatable command-line parameter that lists the game server configs of the servers to start. E.g.

```bash
bundle exec ruby lib/webapp.rb --start 'Server 1' 'Server 2' 'Server 3'
```

Ensure arguments to the start scripts are passed through to the actual entrypoint.

**Comment by Joe-Klauza** [comment]
This is done, but the startup scripts need to be modified to support pass-through to `webapp.rb`.

**Comment by Joe-Klauza** [comment]
Closed by 8c763df.
